features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/metric/networksettled/NetworkSettledMetricResolverTest.kt

```diff
@@ -252,6 +252,33 @@ internal class NetworkSettledMetricResolverTest {
         assertThat(afterViewStoppedMetricValue).isEqualTo(expectedMetricValue)
     }
 
+    @Test
+    fun `M skip last value W resolveMetric(){ view was stopped, last value was drop }`(forge: Forge) {
```

Comment:
"last value was drop" -> "last value was dropped". And the test function seems related with "resource stopped", not "view stopped" if I am not mistaken